core/framework/Response.php

```diff
@@ -535,7 +535,7 @@ public function getFooterDecoration()
          */
         public function setCookie($key, $value, $expiration = 864000)
         {
-            $expiration = ($expiration !== null) ? NOW + $expiration : null;
+            $expiration = ($expiration !== null) ? intval(NOW + $expiration) : null;
```

Comment:
I've seen this crop up before, and I'd like to find out *where* this code is being called with `$expiration` being a string.